src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -12219,7 +12221,14 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
     }
 
     this->TotalCoolingCapacity = TotalCondCoolingCapacity * CoolingPLR;
-    this->TotalHeatingCapacity = TotalCondHeatingCapacity * HeatingPLR;
+    // adjustment for matching HP heating rate and coil heating rate
+    this->TotalHeatingCapacity = TotalCondHeatingCapacity * HeatingPLR * (this->RatedEvapCapacity / (this->RatedEvapCapacity + Pipe_Q_h));
+    if (this->VRFCondPLR < 1.0) {
+        this->TotalHeatingCapacity = TotalCondHeatingCapacity * HeatingPLR * this->PipingCorrectionHeating;
+    }
+    if (this->TUHeatingLoad / this->PipingCorrectionHeating > TotalCondHeatingCapacity) {
+        this->TotalHeatingCapacity = this->HeatingCapacityPrev * HeatingPLR * this->PipingCorrectionHeatingPrev;
+    }
```

Comment:
TotalHeatingCapacity should match the sum of the TU capacity + piping losses. So isn't it just that? TotalHeatingCapacity = Q_h_TU_PL = TU_HeatingLoad + Pipe_Q_h?

Reply:
yes, it is matching this.